uint64 timestamp
string id
Storage storage
uint32 begin
uint32 end
uint32 reward
Item items